mcs_benchmark_data/pipelines/commonsense_qa/kagnet_commonsense_qa_submission_transformer.py

```diff
@@ -17,7 +20,15 @@ class KagnetCommonsenseQaSubmissionTransformer(_BenchmarkSubmissionTransformer):
 
     def transform(
         self,
+        *,
+        extracted_path: Path,
+        file_names: KagnetCommonsenseQaSubmissionFileNames,
         **kwds,
     ) -> Generator[_Model, None, None]:
 
-        yield from self._transform(submission_name="kagnet", **kwds)
+        yield from self._transform(
+            extracted_path=extracted_path,
+            file_names=file_names,
+            submission_name="kagnet",

```

Comment:
Prefer to pass constants like this to the superclass constructor (__init__) rather than passing them around in methods.